package.json

```diff
@@ -11,6 +11,7 @@
     "dependencies": {
         "bluebird": "^2.9.9",
         "lodash": "^3.2.0",
+        "node-tech-logger": "git://github.com/AirVantage/node-tech-logger.git#1.0.0",
```

Comment:
Should we use `v2.x` instead ? I wonder how it really behaves when the module which initializes the _node-tech-logger_ requires the 1.0.0 version and that one of its dependency requires a different version (and it assumes that the logger was initialized before...)